chapters/functions.tex

```diff
@@ -380,15 +366,15 @@ \section{Pure Modelica Functions}\label{pure-modelica-functions}
 
 Examples of restrictions on optimizing pure functions:
 \begin{lstlisting}[language=modelica]
-  Real x=if noEvent(abs(x))<1 then asin(x)  else 0; // Cannot move asin(x) out of then
+  Real x = if noEvent(abs(x)) < 1 then asin(x) else 0; // Cannot move asin(x) out of then
```

Comment:
Have entire comment on next line?

Reply:
I made it several lines, and modified the comment to say "if-branch" instead of "then".

Please mark as resolved if this seems OK.

Reply:
Ok, looks god.